packages/create-astro/src/actions/dependencies.ts

```diff
@@ -25,12 +25,15 @@ export async function dependencies(
 		await spinner({
 			start: `Dependencies installing with ${ctx.pkgManager}...`,
 			end: 'Dependencies installed',
-			while: () =>
-				install({ pkgManager: ctx.pkgManager, cwd: ctx.cwd }).catch((e) => {
-					// eslint-disable-next-line no-console
+			while: () => {
+				return install({ pkgManager: ctx.pkgManager, cwd: ctx.cwd }).catch((e) => {
 					error('error', e);
-					process.exit(1);
-				}),
+					error(
+						'error',
+						'Dependencies failed to install, please install them manually after setup.'
```

Comment:
Could we specify the command users need to run here something like this?

```suggestion
						`Dependencies failed to install, please run "${ctx.pkgManager} install" to install them manually after setup.`
```

Not sure if we usually have a style for code like this? Bold text?